pkgs/by-name/js/jsign/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  makeWrapper,
+  jre_headless,
+  maven,
+  pcsclite,
+  yubico-piv-tool,
+  opensc,
+}:
+
+let
+  jre = jre_headless;
+in
+maven.buildMavenPackage rec {
+  pname = "jsign";
+  # For build from non-release, increment version by one and add -SNAPSHOT
+  # e.g. 7.2-SNAPSHOT
+  version = "7.1";
+
+  src = fetchFromGitHub {
+    owner = "ebourg";
+    repo = "jsign";
+    tag = version;
+    hash = "sha256-+ZErUCTbAI4uzhZGVQ5+awi4N4hnL3RD6SuoNdiXxBs=";
+  };
+
+  # To get libraries from LD_LIBRARY_PATH in addition to /usr/lib
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/ebourg/jsign/pull/306.patch";
+      hash = "sha256-MRM7KsR+oxJTUhkNxzksTVfzNjQ1hWuBQAkx+q2MTNw=";
+    })
+  ];
+
+  mvnHash = "sha256-k/04IHQ90OxSlOzstCTe2QhddZNpqPFsTqkVLjLHArM=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  # The tests try to access the network
+  doCheck = false;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share
+    install -Dm644 jsign/target/jsign-${version}.jar $out/share/jsign.jar
+
+    makeWrapper ${jre}/bin/java $out/bin/jsign \
+      --prefix LD_LIBRARY_PATH : ${
+        lib.makeLibraryPath [
+          yubico-piv-tool
+          opensc
+        ]
+      } \
+      --add-flags "-Dsun.security.smartcardio.library=${lib.getLib pcsclite.lib}/lib/libpcsclite.so.1 -jar $out/share/jsign.jar"
```

```suggestion
      --add-flags "-Dsun.security.smartcardio.library=${lib.getLib pcsclite}/lib/libpcsclite.so.1 -jar $out/share/jsign.jar"
```
That attr should've been removed